Adding the malaria cause model (#1265)
ihmeuw/vivarium_research: docs/source/models/causes/malaria/index.rst

for prevention and treatment, but these are not always accessible. 

An individual with uncomplicated malaria experiences one to two weeks of persistent fever, 
chills/shivering, sweating, joint pains, and headache. The individual will 
likely be lethargic and feverish, causing loss of daily function during the 
attack. Individuals with an untreated P. falciparum infection may develop 
severe malaria, which includes the symptoms of uncomplicated malaria but may 
also involve swelling, difficulty breathing, unconsciousness, and potentially 
death. Microscopy is considered the gold-standard diagnostic approach for the 
purposes of GBD. The relevant ICD-10 codes are B50-B54. [GBD-2019-Capstone-Appendix-Malaria-2020]_

According to the latest world malaria report, there were 247 million cases of 
malaria in 2021 compared to 245 million cases in 2020. The estimated number of 
malaria deaths stood at 619,000 in 2021 compared to 625,000 in 2020.

Four African countries accounted for just over half of all malaria deaths 
worldwide: Nigeria (31.3%), the Democratic Republic of the Congo (12.6%), 
United Republic of Tanzania (4.1%) and Niger (3.9%). [WHO-Malaria-2020]_

> Modeling Malaria in GBD 2020

Reviewer: ```suggestion
Modeling Malaria in GBD 2021
```

----------------------------

> Malaria is modeled separately for inside versus outside sub-Saharan Africa. 

Reviewer: ```suggestion
Malaria is modeled separately for inside versus outside sub-Saharan Africa in GBD 2021. 
```

However, Ethiopia is modeled as outside sub-Saharan Africa since it exhibits 
epidemiological trends and has data availability and quality more akin to non-African 
settings. 

For countries in sub-Saharan Africa (Nigeria in the nutrition optimization model) 
the PfPR surveys by the Malaria Atlas Project (MAP) were used to predict malaria 
rates for both the non-fatal and fatal models. 

For countries outside of sub-Saharan Africa (Ethiopia and Pakistan in this project) surveillance 
systems tend to be stronger and so national and subnational case reports are the 
primary data source for the nonfatal model. 

> Nonfatal modeling in all countries combined data sources to generate pixel-level predictions 

Reviewer: ```suggestion
Nonfatal modeling in all countries combined data sources to generate rasterized predictions
```

> of clinical incidence rates. These predications were combined with high-resolution gridded population 

Reviewer: ```suggestion
of clinical incidence rates  on a 5 km by 5 km grid. These predications were combined with high-resolution gridded population 
```

data to estimate total cases per pixel-year. These were then aggregated to GBD 
national/ subnational areas. 

For the fatal model, MAP data was used to estimate the cause specific mortality rate. 
This was then combined with the incidence rate from the nonfatal model in order to 
find estimates for case fatality rate. This in turn was used to find annual mortality 
rates for each location. [GBD-2019-Capstone-Appendix-Malaria-2020]_

.. todo::

   GBD 2021 includes COVID shocks. We need to decide if and how we want to include this information. 

A systematic review of malaria severity was conducted, from which simple 
severity splits were obtained and applied across all cases:

.. list-table:: Malaria severity splits
	:widths: 5 50 5
	:header-rows: 1
	
	* - Severity level
	  - Lay description
	  - Disability weight
	* - Mild

> 	  - Mild malaria	Has a low fever and mild discomfort but no difficulty with daily activities.	

Reviewer 2: ```suggestion
	  - Mild malaria	
	  - Has a low fever and mild discomfort but no difficulty with daily activities.	
```

> 	  - Mild malaria	Has a low fever and mild discomfort but no difficulty with daily activities.	

Reviewer 2: ```suggestion
	  - Has a low fever and mild discomfort but no difficulty with daily activities.	
```

	  - 0.006 (0.002–0.012)
	* - Moderate
	  - Has a fever and aches and feels weak, which causes some difficulty with daily activities. 
	  - 0.051 (0.032–0.074)
	* - Severe
	  - Has a high fever and pain and feels very weak, which causes great difficulty with daily activities. 
	  - 0.133 (0.088–0.19)

GBD Hierarchy
-------------

.. image:: malaria_cause_hierarchy.svg

Cause Model Diagram
-------------------

.. image:: malaria_cause_model.svg

S: **S**\ usceptible to malaria

I: **I**\ nfected and currently experiencing malaria

Model Assumptions and Limitations
---------------------------------

Malaria has been modeled extensively by other teams. For this model, 
we will not be including any information on vectors and so it is not 
appropriate for interventions targetting malaria prevention or treatment 
directly. 

There is evidence that people living in malaria endemic areas do gain immunity over 
their lifetimes. We assume this is represented in prevalence and incidence rates from 
GBD. We do not include in this model any gains in malaria resistance from prior exposure. 

> 

Reviewer 2: I think this is great to note! For our purposes, we really just care about malaria DALYs attributable to CGF since our model is mainly about CGF and not about malaria transmission dynamics - maybe this would be good additional context to note here about why we don't need to get into any of these additional details.

	  - Simulant currently has malaria

.. list-table:: State Data
	:widths: 5 10 10 20
	:header-rows: 1
	
	* - State
	  - Measure
	  - Value
	  - Notes
	* - S
	  - prevalence
	  - 1-prevalence_I
	  - 
	* - S
	  - birth prevalence
	  - 1 
	  - 
	* - S
	  - emr
	  - 0
	  -
	* - S
	  - disability weight
	  - 0
	  -
	* - I
	  - prevalence
	  - **For early neonatal age group:** (birth_prevalence_I + prevalence_345)/2. **For all other age groups:** prevalence_345
	  - 
	* - I
	  - birth prevalence
	  - 0 
	  - 
	* - I
	  - excess mortality rate
	  - :math:`\frac{\text{deaths_c345}}{\text{population} \,\times\, \text{prevalence_345}}`
	  - 
	* - I
	  - disability weight
	  - :math:`\displaystyle{\sum_{s\in \text{sequelae_malaria}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
	  - Malaria sequelae are: 121, 122, 123
	* - All
	  - cause-specific mortality rate
	  - :math:`\frac{\text{deaths_c345}}{\text{population}}`
	  -

.. list-table:: Transition Data
	:widths: 10 10 10 10 10
	:header-rows: 1
	
	* - Transition
	  - Source State
	  - Sink State
	  - Value
	  - Notes
	* - i
	  - S
	  - I
	  - :math:`\frac{\text{incidence_rate_c345}}{1-\text{prevalence_I}}`

> 	  - Incidence in GBD are estimated for the total population. Here we transform incidence to be a rate within the susceptible population.

Reviewer 2: ```suggestion
	  - Above is equivalent to "load standard data" Vivarium public health function ("suscpetible-population incidence rate). Incidence in GBD are estimated for the total population. Here we transform incidence to be a rate within the susceptible population.
```

The engineers have discussed that it would be helpful if we indicate whether or not they should use the "load standard data" functions for these kind of things. Here, load standard data will directly return what's included in this table, so just adding an additional note about that

	  - Uniform distribution between 14 and 28 days
	  - Obtained from [GBD-2019-Capstone-Appendix-Malaria-2020]_
	  - This value should not vary by age group
	* - incidence_rate_c345
	  - como
	  - Incidence of malaria within the entire population
	  - 
	* - population
	  - demography
	  - Mid-year population for given age/sex/year/location
	  -
	* - prevalence_s{`sid`}
 	  - como
	  - Prevalence of sequela with id `sid`
	  - Sequela used here are 121, 122, and 123 
	* - disability_weight_s{`sid`}
	  - YLD appendix
	  - Disability weight of sequela with id `sid`
	  - Sequela used here are 121, 122, and 123 

.. list-table:: Restrictions
	:widths: 15 15 20
	:header-rows: 1

	* - Restriction type
	  - Value
	  - Notes
	* - Male only
	  - False
	  -
	* - Female only
	  - False
	  -
	* - YLL only
	  - False
	  -
	* - YLD only
	  - False
	  -
	* - YLL age group start
	  - Early neonatal
	  - age_group_id = 2; [0-7 days)
	* - YLL age group end
	  - 95 plus
	  - age_group_id = 235; 95 years +
	* - YLD age group start
	  - Early neonatal
	  - age_group_id = 2; [0-7 days)
	* - YLD age group end
	  - 95 plus
	  - age_group_id = 235; 95 years +

Validation Criteria
-------------------

> Simulation results should replicate the GBD 2020 cause-specific mortality rate, 

Reviewer: ```suggestion
Simulation results should replicate the GBD 2021 cause-specific mortality rate, 
```
